### PR TITLE
update readme for 0.5.0

### PR DESCRIPTION
**Pull request by a-zorina**

### CODE REVIEW

**Review by melsomino — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -179,6 +203,49 @@ tondev sol set --compiler 0.38.0 --linker 0.23.54
...
+This command creates a basic C++ contract with comments that you can observe and compile.
+
+```shell
+tondev clang create Contract
+```
+
+#### Compile
+
+This command compiles and links a selected C++ contract.
+After successful compilation you get .abi.json and .tvc files that you can later [use in your DApps to deploy and call contract methods](https://docs.ton.dev/86757ecb2/p/07f1a5-add-contract-to-your-app-/b/462f33).
```
[the remark this answers is not in the document]
Comment (a-zorina) [reply README.md]: fixed in two places

File: README.md [hunk 2]
```diff
@@ -356,89 +423,431 @@ tondev tonos-cli update
...
+
+This command deletes a previously added signer from signer registry.
+
+```bash
+tondev signers delete signer_name
+```
 
 ### Network Registry
 
 Networks registry is a convenient way to organize all of your network configurations in one place.
```
Comment (elasticLove1) [inline README.md]: Rename `networks` command to `network` everywhere too

File: README.md [hunk 3]
```diff
@@ -356,89 +423,431 @@ tondev tonos-cli update
...
 
-#### Add Command
+#### Add a network
 
-Add network with specified endpoints to the registry.
+This command adds a network to the tondev registry.
 
-#### Delete Command
+```bash
+tondev networks add network_name network_endpoint
```
Comment (elasticLove1) [inline README.md]: Add one more sample where you add several endpoints for the same network - for more reliability, for instance
add `mainnet` network with main2.ton.dev, main3.ton.dev, main4.ton.dev endpoints.

File: README.md [hunk 4]
```diff
@@ -356,89 +423,431 @@ tondev tonos-cli update
...
+```bash
+$ tondev contract info Contract
+
+Configuration
+
+  Network: main
+  Signer:  sign1
+
+Address: 0:0435cb4e70585759ac514bb9fd1770caeb8c3941d882b5a16d589b368cb49261
+Account: Not exists
```
Comment (elasticLove1) [inline README.md:647]: old incorrect text

File: README.md [hunk 5]
```diff
@@ -356,89 +423,431 @@ tondev tonos-cli update
...
+
+Deploying...
+```
+
+Network, signer and account address parameters can be overridden and constructor parameters specified in the deploy command with the following options:
+
+```bash
+$ tondev contract deploy -h
+TONDev Version: 0.5.0
+Use: tondev contract deploy file function [options]
```
Comment (elasticLove1) [inline README.md:701]: Add a sample of multisig deploy with many custodians in one line - we need to show that tondev does not require any {} symbols and that it is easier than tonos-cli console input

File: README.md [hunk 6]
```diff
@@ -356,89 +423,431 @@ tondev tonos-cli update
...
+  1) func1
+  2) func2
 
-Load and emulates executor for the specified contract on the local TVM.
+  Select function (number): 2
+
+Running...
+```
+
+Network, signer and account address parameters can be overridden and function parameters specified in the command with the following options:
```
Comment (elasticLove1) [inline README.md:743]: also need to add a sample of multisig usage, submit transaction for instance

**Review by elasticLove1 — CHANGES_REQUESTED** [review]

**Pull request merged**